docs/platform/release-notes/tizen-7-0-m2.md

```diff
@@ -0,0 +1,543 @@
+  - The C/C# API to provide TTS service states such as ready, synthesizing, and playing to TTS clients has been added.
+- Voice Control Framework
+  - The IPC interface between the voice control engine and voice control clients has been changed from D-Bus to TIDL, which is a proper IPC method between applications.
+  - The C API to reduce background volume when the voice manager client wants has been added.
+- Multi-Modal Framework
+  - The new voice-touch feature has been added.
+    - The screen analyzer as one of the input modalities has been added.
+    - The voice recognition input modality has been added.
+    - IU (Intent Understanding) Module for the voice touch has been enhanced.
+    - Always listening mode has been supported.
+- Enlightenment
+  - The ratio-fit resize has been added. It provides that a window can be resized with a ratio-fit value.
+  - The done event for interactive move and resize has been added. A wayland client can know when the interactive move and resize has been done.
+
+#### Fixes
+
+- Enlightenment
+  - Several issues on drag and drop have been fixed.
+    - An issue of hiding a dragged window intermediately has been fixed.
+    - An issue of misusing a hash function has been fixed.
+    - An issue of sending a drop event when the drop is cancelled has been fixed.
+
+
+### Graphics and UI
+
+#### New and changed features
+
+- Rendering
+  - Support for a dedicated UI thread has been added.
+  - Support for EGL_EXT_image_dma_buf_import has been added.
+- Scene3D
+  - Tizen.NUI.Scene3D namespace has been added.
+  - SceneView and Model have been added.
+  - Support for facial animation has been added.
+  - Support for cubemap images has been added.
+  - BVH loader has been added.
+- View
+  - Various properties about touch and key event have been added.
+- Input and Gesture
+  - Pan gesture behavior has been optimized.
+- Images
+  - Support for Lottie dynamic properties has been added.
+  - Support for asynchronous SVG file loading has been added.
+  - Support for asynchronous Lottie file loading has been added.
+  - Support for GPU alpha masking has been added.
+- Performance Improvement
+  - Some image operations have been optimized.
+  - Texture cache logic has been optimized.
+  - Building time of a uniform map has been decreased.
+  - Signal connection time has been decreased.
+  - Vector operation time has been decreased.
+  - Text rendering time has been decreased.
+- Rive-Tizen
+  - rive-cpp code base to support Rive 2.0 API has been updated.
+  - Skia based animation renderer has been introduced.
+- EFL
+  - Webp animation memory usage has been optimized.
+  - Stability has been improved.
+- Aurum
+  - Support for intelligent UI automation has been added.
+  - Support for the repeated key event has been added.
+  - Support for the interface to get text minimum rectangle has been added.

```

Comment:
I am a bit confused regarding what *text minimum rectangle* means

Reply:
it means the area occupied by rendered texts.